All project code/views/partials/menu.ejs

```diff
@@ -1,18 +1,21 @@
 <nav class="navbar navbar-expand-lg bg-light">
   <div class="container-fluid">
-      <a class="navbar-brand" href="/main">AirfareAffairs</a>
+      <a class="navbar-brand" href="/landing">AirfareAffairs</a>
       <ul class="navbar-nav me-auto" id="navbar-list">
         <li class="nav-item">
           <a class="nav-link" href = "/main">Home</a>
         </li>
         <!-- <li class="nav-item">
           <a class="nav-link" href = "/search">Search</a>
         </li> -->
+        <li class="nav-item">
+          <a class="nav-link" href = "/profile">Profile</a>
+        </li>
         <li class="nav-item">
           <a class="nav-link" href = "/searchWeather">Search Weather</a>
         </li>
         <li class="nav-item">
-          <a class="nav-link" href = "/profile">Profile</a>
+          <a class="nav-link" href = "/flights">Flights</a>
```

Comment:
I think Profile has a different purpose, like displaying data on the user from the database (like, which destinatioins the user added to their account)? Maybe instead you should make an additional navbar item.